hunt/lib/tasks/import.rake

```diff
@@ -11,9 +11,26 @@ namespace :import do
      # Open animals spreadsheet
     data = Roo::Spreadsheet.open('lib/data/animals.xlsx')
     headers = data.row(1)
-    puts "The data headers are: #{headers}"
+    #puts "The data headers are: #{headers}"
 
     # For each row in the file, create a new animal below:
 
+    num_animals = 0
+
+    data.each_with_index do |row, index|
+      next if index == 0
+
+      if Animal.exists?(state: row[0].to_s.titleize,
+        name: row[1].to_s.titleize, category: row[2].to_s)
+        next
+      end
+
+      Animal.create!(state: row[0].to_s.titleize,
```

Comment:
Oh I didn't think about this in the original AC, but it's probably a good idea to add: Because we're using `create!` here, if any animal fails validations, this whole process will stop. What we probably want to do instead is print a message to the user letting them know which animal failed (you can just print the row with a message), and then continue to import the rest of the animals. 

using `new`/`save` might be helpful here since `save` returns whether or not the record was successfully saved to the database.